setup.py

```diff
@@ -27,7 +27,8 @@
         "scikit-image<0.15",
         "scipy==1.2.2",
         "shapely==1.6.4",
-        "tensorflow-gpu==1.15.4",
+        "tensorflow==2.0.0",
```

Comment:
Do we need to install both `tensorflow` and `tensorflow-gpu`?

Reply:
I think we only need tensorflow-gpu. Tensorflow installs the CPU version.